src/index.js

```diff
@@ -439,6 +469,17 @@ class ServerlessAppsyncPlugin {
           Tags: !config.tags ? undefined : this.getTagsConfig(config),
         },
       },
+      ...(config.authenticationType === 'AWS_LAMBDA' && {
+        [`${logicalIdGraphQLApi}LambdaAuthorizerPermission`]: {
+          Type: 'AWS::Lambda::Permission',
+          Properties: {
+            Action: 'lambda:InvokeFunction',
+            FunctionName: this.getLambdaAuthorizerConfig(config).AuthorizerUri,
```

Comment:
Here too, you probably can use something like `getLambdaArn(config.lambdaAuthorizerConfig)`